.drone.yml

```diff
@@ -100,6 +100,9 @@ steps:
   image: 906087756158.dkr.ecr.us-east-1.amazonaws.com/r-dev-ci-mpn-4.0:2020-06-08
   commands:
   - R -s -e 'devtools::install_deps(upgrade = '"'"'never'"'"')'
+  # nmrec isn't available until the 2023-09-19 snapshot.
+  - git clone --depth 1 https://github.com/metrumresearchgroup/nmrec.git /tmp/nmrec
+  - R -s -e 'devtools::install("/tmp/nmrec", upgrade = "never")'
```

Comment:
I think we want to change these lines before merging FYI

Reply:
> I think we want to change these lines before merging FYI

What is your suggested change?

Reply:
As the comment on line 103 says, nmrec isn't available on this image's (`r-dev-ci-mpn-4.0:2020-06-08`) snapshot, so we need to keep the custom install.  We could update the custom command to install nmrec from the 2023-09-19 snapshot instead of the Git repo (or equivalently just make this use the v0.2.0 tag from the nmrec repo).  

I could see the argument that pinning the nmrec version is more inline with the "test oldest" goal of this build.  Is that the reason for your suggestion?

Reply:
> I could see the argument that pinning the nmrec version is more inline with the "test oldest" goal of this build

a92c9bf4 pins the nmrec version.

Reply:
>I could see the argument that pinning the nmrec version is more inline with the "test oldest" goal of this build. Is that the reason for your suggestion?

I thought about that too, but mainly it was just to reduce some of the variability in what version of `nmrec` we pull in drone. In the event `nmrec` and `bbr` were undergoing larger refactors, I figured it would be good to have a more stable version of `nmrec` rather than whatever was on `main` at the time. Pointing to a specific version definitely takes care of that and IMO is actually better than pointing to an MPN snapshot anyways (so we can track `nmrec` and `bbi` versions similarly).

Reply:
> mainly it was just to reduce some of the variability in what version of `nmrec` we pull in drone.

Having the nmrec version be different across the pipelines is fine, and in many cases desirable because it increases what you're testing.  On average nmrec's main should be more stable than a release because it has fixes, but in the event that it does break bbr compatibility, we want to know because it shouldn't.

In any case, for the oldest build using an old nmrec for consistency as a92c9bf works for me, so we're set for the purposes of this PR.